try-cb-dotnet/wwwroot/index.html

```diff
@@ -0,0 +1,6 @@
+<h1>.NET Travel Sample API</h1>
+    A sample API for getting started with Couchbase Server and the .NET SDK.
+    <ul>
```

Comment:
Indentation for the `<ul>` block seems weird here actually, maybe something like:
```
<h1>.NET Travel Sample API</h1>
A sample API for getting started with Couchbase Server and the .NET SDK.
<ul>
   <li><a href="/apidocs">Learn the API with Swagger, interactively</a>
   <li><a href="https://github.com/couchbaselabs/try-cb-dotnet">GitHub</a>
</ul>
```